### ISSUE TITLE
Inode false positive message after upgrade to Debian 8

### ISSUE THREAD

**Issue by spridarn** [munin-monitoring/munin#561]
I upgraded to Debian 8 for some months ago and since then my Munin server staarted to spam me with a lot of mail regarding Inode. I havn´t changed anything since before the upgrade and before the upgrade I didn´t recevie this messages. 

 I accept if you got some warning och critical message but all this Inode mails are with status OK. One red thread when I got this mail is when I access one of my machines with SSH. Some minute after that I got this message. An example of the message I got is below:

mydomain.se :: web01.mydomain.se :: Disk usage in percent
	OKs: /mnt/webdata is 21.02, /run/user/0 is 0.00, /run/lock is 0.00, /dev/shm is 0.00, / is 49.67, /run is 7.14, /sys/fs/cgroup is 0.00, /mnt/owncloud is 3.91.

mydomain.se :: web01.mydomain.se :: Inode usage in percent
	OKs: / is 2.07, /run is 0.10, /run/lock is 0.00, /dev/shm is 0.00, /sys/fs/cgroup is 0.00, /mnt/owncloud is 0.02, /dev is 0.07, /mnt/webdata is 0.00, /run/user/0 is 0.00.

I can´t understand what cause this problem, anyone having an idea?

This is my environment for the Munin server:

Distributor ID: Debian
Description:    Debian GNU/Linux 8.2 (jessie)
Release:        8.2
Codename:       jessie

This is munin-node-configure (munin-node) v2.0.25-1

**Comment by ssm** [comment]
I wonder, is it flapping between UNKNOWN, which is not reported, and OK?

**Comment by spridarn** [comment]
No it´s always OK only so it´s not a mail that the status has changed

**Comment by ssm** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=788736 looks to be the same problem.

Cause: filesystems are added and removed in /run/user/ as users log in and out. This triggers munin-limits.

Solution: Configure the df and df_inode plugins to ignore them.  

Please try adding an exclusion for those filesystems in the plugin config in /etc/munin/plugin-conf.d/munin-node, under the [df*] header, like:

    [df*]
    env.exclude_re ^/run/user
    # […] other env.settings

**Comment by ssm** [comment]
Closing this issue.  The configuration file in question is specific to the Debian packaging

**Comment by spridarn** [comment]
Thanks, changed my config files at friday and havn't got any mail during the weekend

**Comment by vadz** [comment]
I'm surprised this was closed as being Debian-specific as I don't see what exactly is Debian-specific here? AFAICS the bug will happen on any system using SystemD, as [link above](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=788736) explains, it happens whenever a filesystem is unmounted and then mounted again.

This definitely looks like a Munin bug to me, why should it generate an "OK" notification if there had never been any warnings nor critical errors before? It's _not_ an error to temporarily unmount a filesystem.

**Comment by ssm** [comment]
The configuration file mentioned in this bug report has its source in the debian packaging, and does not come from here. That's the "debian specific" part.

**Comment by ssm** [comment]
Also, munin-limits reports all state changes, not errors. A new file system is a state change.

**Comment by vadz** [comment]
I don't want to waste your time if you're really sure you don't want to change anything in munin, but I just don't understand at all what you're saying, so let me try to explain how I see things once again just in case there is a misunderstanding:

1. The configuration file is used as a place for the bug workaround. The bug itself is completely independent of this configuration file, so I'm at a loss what does it have to do with the problem.
2. The fact that `df` plugin reports new file system as state change is indeed a bug as it makes it unusable in practice on any multiuser system using systemd. Any reasonable user of munin expects the `df` plugin to complain only about running out of space on the volume and this is obviously the intention of the code (e.g. it doesn't send any notifications if neither "warning" nor "critical" thresholds are set).

*TL;DR*: Munin df plugin doesn't behave as expected on any platform using systemd, this has nothing to do with Debian.
